_data/merchants.yml

```
@@ -79,6 +81,8 @@
       url: https://www.morphtoken.com/
     - name: Newton - Canadian cryptocurrency exchange
       url: https://newton.co
+    - name: Pay2day.top (Ru) - Buy and sell cryptocurrency
```

```suggestion
    - name: PAY2DAY.TOP (RU) - Buy and sell cryptocurrency
```
Not sure that it's necessary, but the business name they provided was in all caps.

---

They use both versions, so i preferred to use the lowercase version.